.github/workflows/docker.yaml

```diff
@@ -0,0 +1,36 @@
+name: Build CLI Docker image
+
+on:
+  release:
+    types: [published]
+  push:
+    branches:
+      - "test_dockerfile"
+
+jobs:
+  push_to_registry:
+    name: Publish docker image
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        golang: [1.19]
+    steps:
+      - name: Check out the repo
+        uses: actions/checkout@v3 
+      - name: Install Go
+        uses: actions/setup-go@v3
+        with:
+          go-version: ${{ matrix.golang }}
+      - name: Build
+        run: make build-all                      
+      - name: Build Image
+        run: docker buildx build -t atlas-cli --platform=linux/amd64 .
+      ## Disabled for now
+      # - name: Login
+      #   run: docker login -u="${{secrets.CONTAINER_USER}}" -p="${{secrets.CONTAINER_SECRET}}" quay.io
+      # - name: Push To quay.io
+      #   id: push-to-quay
+      #   run: docker push quay.io/mongodb/atlas-cli
```

Comment:
Let's omit this github workflow file from this PR
We can introduce it in a future follow-up story

Reply:
Removed